snowflake/connect.go

```diff
@@ -82,6 +82,16 @@ func connect(ctx context.Context, d *plugin.QueryData) (*sql.DB, error) {
 	if config.OAuthAccessToken != nil {
 		oauthAccessToken = *config.OAuthAccessToken
 	}
+	// TODO - Required to query general tables through steampipe
```

Comment:
Is this something we're planning on adding soon? What's the result of us not having arg support for these yet?